### PR TITLE
Include AutoMod for Member Profile actions

### PR DESCRIPTION
**Pull request by Doc94**
**Description:** Include Changes in AutoMod related to Member Profiles

**Justification:** https://github.com/discord/discord-api-docs/commit/60a711d841394492fc3a36c8d14c19fea00c2ccd

### CODE REVIEW

File: core/src/main/java/discord4j/core/object/automod/AutoModRule.java [hunk 1]
```diff
@@ -305,10 +315,12 @@ public int getValue() {
          * @return The type of message.
          */
         public static AutoModRule.EventType of(final int value) {
-            switch (value) {
-                case 1: return MESSAGE_SEND;
-                default: return UNKNOWN;
+            for (AutoModRule.EventType eventType : values()) {
```
Comment (Azn9) [inline core/src/main/java/discord4j/core/object/automod/AutoModRule.java:318]: I think that to follow the library standards, you should keep a switch here

**Review by Azn9 — CHANGES_REQUESTED** [review]

File: core/src/main/java/discord4j/core/object/automod/AutoModRule.java (hunk 1 above)
Comment (Doc94) [reply core/src/main/java/discord4j/core/object/automod/AutoModRule.java:318]: Yeah but this logic are in another places too, like others enums in the AutoMod thing

Comment (Azn9) [reply core/src/main/java/discord4j/core/object/automod/AutoModRule.java:318]: We should choose one logic and apply it everywhere I think, to keep everything the same.
 And switch statements are more efficient so I would privilege them.

Comment (Doc94) [reply core/src/main/java/discord4j/core/object/automod/AutoModRule.java:318]: by the moment i keep this here and in the future mark a standar for this things.

Comment (Azn9) [reply core/src/main/java/discord4j/core/object/automod/AutoModRule.java:318]: LGTM, personally I use the switch variant on all of my PRs

**Pull request merged**